apps/server/src/modules/deletion/uc/deletion-request.uc.spec.ts

```diff
@@ -1,25 +1,25 @@
 import { Test, TestingModule } from '@nestjs/testing';
 import { DeepMocked, createMock } from '@golevelup/ts-jest';
 import { setupEntities } from '@shared/testing';
-import { AccountService } from '@modules/account/services';
+import { AccountService } from '@modules/account';
 import { ClassService } from '@modules/class';
-import { CourseGroupService, CourseService } from '@modules/learnroom/service';
-import { FilesService } from '@modules/files/service';
-import { LessonService } from '@modules/lesson/service';
+import { CourseGroupService, CourseService } from '@modules/learnroom';
```

Comment:
I already mark it for me in api-modules file. I must think about it after lunch.

Reply:
So what to do with it?

Reply:
waiting for answer

Reply:
It is fine for now.